content/blog/zephyrs-summer-intern-experience-with-pulumi/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Zephyr's Summer Intern Experience with Pulumi"
+date: "2020-09-17"
+meta_desc: "Zephyr's internship experience in Summer 2020, personal growth, skills learned and reflection"
+meta_image: zephyr-pulumi.png
+authors: ["zephyr-zhou"]
+tags: ["pulumi","intern"]
+---
+
+Hi, I am [Zephyr Zhou](https://www.linkedin.com/in/zephyr-zhou-a17741196/), a senior Computer Science student at the University of Washington. I have just spent this entire Summer interned at Pulumi. This is my first internship ever in my life. Thanks to Pulumi for providing this opportunity even in this difficult time of the Covid-19 epidemic. Despite the sad truth that I couldn't get in touch offline, I believe this would be one of the most precious memory.
```

Comment:
```suggestion
Hi, I am [Zephyr Zhou](https://www.linkedin.com/in/zephyr-zhou-a17741196/), a senior Computer Science student at the University of Washington. I spent this past summer interning at Pulumi. This is my first internship ever in my life. Thanks to Pulumi for providing this opportunity even in this difficult time of the Covid-19 epidemic. Despite the sad truth that I couldn't get in touch offline, I believe this will be one of my most precious memories.
```

Reply:
Thanks for the kind words! We're glad this will be a lasting memory for you!